README.md

```diff
@@ -167,149 +168,24 @@ For major changes, you are welcomed to open an issue and discuss what you would
-            💻
-            </a>
-        </td>
-        <td align="center">
-            <a href="https://github.com/Aniket31-coder">
-            <img src="https://avatars3.githubusercontent.com/u/55101835?v=4" width="100px" alt=""/><br />
-            <sub><b>Aniket31-coder</b></sub>
-            </a><br />
-            <a href="https://github.com/Tejas1510/Athavani/commits?author=Aniket31-coder">
-            💻
-            </a>
-        </td>
-        <td align="center">
-            <a href="https://github.com/Ashuto7h">
-            <img src="https://avatars1.githubusercontent.com/u/53266261?v=4" width="100px" alt=""/><br />
-            <sub><b>Ashuto7h</b></sub>
-            </a><br />
-            <a href="https://github.com/Tejas1510/Athavani/commits?author=Ashuto7h">
-            💻
-            </a>
-        </td>
-        <td align="center">
-            <a href="https://github.com/musavveer">
-            <img src="https://avatars2.githubusercontent.com/u/62888562?v=4" width="100px" alt=""/><br />
-            <sub><b>musavveer</b></sub>
-            </a><br />
-            <a href="https://github.com/Tejas1510/Athavani/commits?author=musavveer">
-            💻
-            </a>
-        </td>
-        <td align="center">
-            <a href="https://github.com/koolgax99">
-            <img src="https://avatars0.githubusercontent.com/u/55532999?v=4" width="100px" alt=""/><br />
-            <sub><b>koolgax99</b></sub>
-            </a><br />
-            <a href="https://github.com/Tejas1510/Athavani/commits?author=koolgax99">
-            💻
-            </a>
-        </td>
-        <td align="center">
-            <a href="https://github.com/SuNiL-Chau">
-            <img src="https://avatars0.githubusercontent.com/u/46418190?v=4" width="100px" alt=""/><br />
-            <sub><b>SuNiL-Chau</b></sub>
-            </a><br />
-            <a href="https://github.com/Tejas1510/Athavani/commits?author=SuNiL-Chau">
-            💻
-            </a>
-        </td>
-        <td align="center">
-            <a href="https://github.com/kenkirito">
-            <img src="https://avatars1.githubusercontent.com/u/68530218?v=4" width="100px" alt=""/><br />
-            <sub><b>kenkirito</b></sub>
-            </a><br />
-            <a href="https://github.com/Tejas1510/Athavani/commits?author=kenkirito">
-            💻
-            </a>
-        </td>
-       <td align="center">
-            <a href="https://github.com/himanshu007-creator">
-            <img src="https://avatars.githubusercontent.com/u/65963997?s=460&u=da0f5f134520e08a42e768efb23942ecd9a25434&v=4" width="100px" alt=""/><br />
-            <sub><b>Himanshu</b></sub>
-            </a><br />
-            <a href="https://github.com/Tejas1510/Athavani/commits?author=himanshu007-creator">
-            💻
-            </a>
-        </td>
-    </tr>
-</table>
+<a href="https://github.com/Tejas1510"><img src="https://github.com/Tejas1510.png?size=50"></img></a>
```

Comment:
Here, you don't have to add manually.
Use this.
https://contrib.rocks/preview

Reply:
Okay, I will make the change